src/main/java/org/gridsuite/voltageinit/server/service/parameters/VoltageInitParametersService.java

```diff
@@ -129,27 +155,42 @@ private void setLowVoltageLimit(List<VoltageLimitOverride> specificVoltageLimits
+    private static CounterToIncrement generateHighVoltageLimit(List<VoltageLimitOverride> specificVoltageLimits,
+                                                               Map<String, VoltageLimitEntity> voltageLevelModificationLimits,
+                                                               Map<String, VoltageLimitEntity> voltageLevelDefaultLimits,
+                                                               boolean isHighVoltageLimitModificationSet,
+                                                               boolean isHighVoltageLimitDefaultSet,
+                                                               VoltageLevel voltageLevel) {
         if (!Double.isNaN(voltageLevel.getHighVoltageLimit()) && isHighVoltageLimitModificationSet) {
-            specificVoltageLimits.add(new VoltageLimitOverride(voltageLevel.getId(), VoltageLimitOverride.VoltageLimitType.HIGH_VOLTAGE_LIMIT, true, voltageLevelModificationLimits.get(voltageLevel.getId()).getHighVoltageLimit()));
+            specificVoltageLimits.add(new VoltageLimitOverride(voltageLevel.getId(), VoltageLimitType.HIGH_VOLTAGE_LIMIT, true, voltageLevelModificationLimits.get(voltageLevel.getId()).getHighVoltageLimit()));
+            return CounterToIncrement.MODIFICATION;
         } else if (Double.isNaN(voltageLevel.getHighVoltageLimit()) && isHighVoltageLimitDefaultSet) {
-            specificVoltageLimits.add(new VoltageLimitOverride(voltageLevel.getId(), VoltageLimitOverride.VoltageLimitType.HIGH_VOLTAGE_LIMIT, false, voltageLevelDefaultLimits.get(voltageLevel.getId()).getHighVoltageLimit() + (isHighVoltageLimitModificationSet ? voltageLevelModificationLimits.get(voltageLevel.getId()).getHighVoltageLimit() : 0.)));
+            specificVoltageLimits.add(new VoltageLimitOverride(voltageLevel.getId(), VoltageLimitType.HIGH_VOLTAGE_LIMIT, false, voltageLevelDefaultLimits.get(voltageLevel.getId()).getHighVoltageLimit() + (isHighVoltageLimitModificationSet ? voltageLevelModificationLimits.get(voltageLevel.getId()).getHighVoltageLimit() : 0.)));
+            if (isHighVoltageLimitModificationSet) {
+                return CounterToIncrement.BOTH;
+            } else {
+                return CounterToIncrement.DEFAULT;
+            }
+        } else {
+            return CounterToIncrement.NONE;
         }
     }
 
     @Transactional(readOnly = true)
     public OpenReacParameters buildOpenReacParameters(VoltageInitRunContext context, Network network) {
-        AtomicReference<Long> startTime = new AtomicReference<>();
-        startTime.set(System.nanoTime());
+        AtomicReference<Long> startTime = new AtomicReference<>(System.nanoTime());
+        final Reporter reporter = context.getRootReporter().createSubReporter("OpenReacParameters", "OpenReac parameters", Map.of(
```

Comment:
OpenReac is a model and is not shown for now to the user.
Rename this subreporter "VoltageInit parameters". This also matches the parent reporter which is named "VoltageInit"